jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/DiagnosticUtils.java

```diff
@@ -125,6 +263,52 @@ public class DiagnosticUtils {
 	 */
 	public static final String DIAGNOSTICS_OPTION_SEPARATOR = ","; //$NON-NLS-1$
 
+/*[IF JFR_SUPPORT]*/
+	/**
+	 * Help information for JFR Java 17 commands
+	 */
+/*[IF JAVA_SPEC_VERSION == 17]*/
+	public static final String printJFRHelp(String helpcommand) {
+		String helpMessageString = null;
+		try {
+			Class<?> clazz;
+			if (helpcommand.equals("JFR.start")) {
+				clazz = Class.forName("jdk.jfr.internal.dcmd.DCmdStart");
+			} else if (helpcommand.equals("JFR.stop")) {
+				clazz = Class.forName("jdk.jfr.internal.dcmd.DCmdStop");
+			} else if (helpcommand.equals("JFR.dump")) {
```

Comment:
The constants `DIAGNOSTICS_JFR_START`, etc. should be used instead of hard-coded string literals here.

Reply:
Thanks, Addressed.